src/kallisto_quant.py

```diff
@@ -29,6 +32,13 @@
 logger.addHandler(filehandler)
 
 
+def choices(population, k):
```

Comment:
Do you need this function in this file?  If needed in concatenate_files only, perhaps just define it in that function, or use tempfile module if that works just as well.

Reply:
This indeed is not needed here.

Reply:
This was sloppiness. Not needed here.